internal/test/go.mod

```diff
@@ -3,9 +3,11 @@ module github.com/openshift/windows-machine-config-bootstrapper/internal/test
 go 1.12
 
 replace (
-	k8s.io/api => k8s.io/api v0.0.0-20190409021203-6e4e0e4f393b
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
-	k8s.io/client-go => k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
+	github.com/openshift/api => github.com/openshift/api v0.0.0-20200205145930-e9d93e317dd1 // OpenShift 4.3
```

Comment:
Wasn't this done in #146 and #151?

Reply:
This was done to the WNI's dependencies, not the framework's.